Bio/SeqUtils/MeltingTemp.py

```diff
@@ -19,10 +19,12 @@
    additives are available.
 
 General parameters for most Tm methods:
- - seq -- A Biopython sequence object or a string.
+ - seq -- A Biopython sequence object, SeqRecord or a string.
  - check -- Checks if the sequence is valid for the given method (default=
-   True). In general, whitespaces and non-base characters are removed and
-   characters are converted to uppercase. RNA will be backtranscribed.
+   True). Whitespaces and non-alphabetic characters are removed and
+   characters are converted to uppercase. RNA will be backtranscribed. When true,
+   if the sequence contains letters that are not valid for the method, it raises
+   a ValueError.
```

Comment:
If check is `True`, does this function raise a `ValueError` if the sequence contains whitespaces or non-alphabetic characters?